### ISSUE TITLE
Sent Connection message, won't connect

### ISSUE THREAD

**Issue by Skarmy** [sunshinekitty/TwitchPlaysPokemon#9]
Recently when I've launched the bot I get the message "Sent Connection Message to Channel Smarmyskarmy" and then it stops there.

I have been running the application perfectly fine until about a day and a half ago. I tried deleting the bot completely and redownloading it, and changing my AUTH code, but it's still not connecting.

My twitch connects to my IRC just fine and when I open mIRC I can see messages coming from my twitch channel, but it won't connect on the bot.

**Comment by Skarmy** [comment]
I got it working, but I don't know which of these things got it working. First I installed an earlier version of your bot, then I changed my AUTH password and refreshed everything, then launched it trough CMD instead of in the folder, I ran the set up using my new password and finally it connected.

I would think changing my AUTH would have been the most obvious thing to have fixed it, but I've done this several times before and it didn't connect me.
